src/app/components/PlaceButton/PlaceButton.tsx

```diff
@@ -14,7 +14,11 @@ function PlaceButton({ place, onPlaceChange }: PlaceButtonProps): JSX.Element {
   }
 
   return (
-    <button className={styles.button} onClick={() => handleClick()}>
+    <button
+      className={styles.button}
+      onClick={() => handleClick()}
+      type="button"
```

Comment:
Is the type of button needed here? (Im not sure 😅)

Reply:
Yes, it is otherwise, the button will submit my form before I want it.

type
The default behavior of the button. Possible values are:
submit: The button submits the form data to the server. This is the default if the attribute is not specified for buttons associated with a <form>, or if the attribute is an empty or invalid value.